pkg/protocol/bthrift/test/kitex_gen/test/k-consts.go

```diff
@@ -0,0 +1,4 @@
+package test
```

Comment:
please add license header for all new files

Reply:
for gen code, please add ignore in .licenserc.yaml. 
and write a script to update the gen code,